src/test/resources/features/SalesForceLeads.feature

```diff
@@ -1,25 +1,67 @@
 Feature: Salesforce Leads
 
-  @Leads
+
   @salesforce @Leads @Web
   Scenario: Create Leads with Open Status
   and Change Lead Status to Contacted or
   Close Leads if not required to Convert
+  Create New or  Select Existing Accounts, Contacts, and Opportunities
+  Change Opportunity Status Qualifcation to Proposal/Quote
+  Create and Link Quote with Opportunity, Account, Contact
+  Quote  Edit Lines (Select Add Discount), (Add Product)
+  Verify Product Added in the Opportunity
+  Change Opportunity Status to Close  Won after Quote Approval
+  Edit Approved Quote to check mark as Ordered & Create Contract
+  Verify Ordered and Contract created after Quote checked as Ordered
+  Activate Order (Change Status Draft to Activate )
+
+
     Given navigate to Salesforce as "Leads Admin"
     And login to Salesforce with "Leads Admin" credentials
     When Click on Hamburger Menu
     And click on Search <"Leads">
     Then verify Leads Page
     When Click on New Button
     And Enter LastName and CompanyName
-    When Click On Save Button
+    And Click on Save Button
     Then verify New with Open Status
     When Click on Converted Tab
     And Click on Select Convert Status
     And Click on Convert button
     Then verify Your lead has been converted
+    When Click on Opportunity Tab in converted popUp
```

Comment:
please follow a pattern through out in the file, either start the steps with small letter or with capital letter

Reply:
Updated

Reply:
and pushed the updated code, please review